fabric/src/main/resources/minescript.py

```diff
@@ -786,6 +786,53 @@ def screen_name():
   """
   return CallScriptFunction("screen_name")
 
+def container_get_item(slot:int):
```

Comment:
As I mention in Minescript.java, this seems redundant with `container_get_items()`.